connectionService.js

```diff
@@ -1,5 +1,5 @@
 import EventEmitter from "eventemitter3";
-import { AsyncStorage } from "react-native";
+import { AsyncStorage } from "@react-native-community/async-storage";
```

Comment:
```suggestion
import { AsyncStorage } from "react-native";
```

Let's not add the external dependency.